.github/workflows/build-dotnet.yml

```diff
@@ -3,21 +3,31 @@ name: Test Build Plugin
 on:
   push:
     branches: [ master ]
+    paths-ignore:
+      - '.github/**'
```

Comment:
Would this not prevent the tests getting ran when the only change is the tests themselves, such as this PR?

Reply:
that is correct, yet since the "Tests" are C# tests it makes no sense to run on changes within the .github folder. (IMHO at least :wink: )

Reply:
I think it makes sense to re-run tests when ever files that affect the outcome of the tests are touched, this of course includes the commands getting ran in the tests, and their configuration.

Reply:
:thinking: I have not thought of it this way yet (for some irrational reason). I have to agree with you. (want me to submit a PR changing it back and only ignore .md changes?)